README.md

```diff
@@ -49,12 +49,12 @@ You can easily build Scylla Operator in your environment:
 
 ## Contributing
 
-We would love for you to contribute to Scylla Operator, help make it even better and learn together! Use these resources to help you get started:
+We would **love** you to contribute to Scylla Operator, help make it even better and learn together! Use these resources to help you get started:
```

Comment:
Isn't a "for" missing in "... **love** for you to ..."?

Reply:
no it is not needed

Reply:
Oh really? Not being native english speaker this really is a learning moment! :) Let's merge then!